src/relay/backend/vm/compiler.cc

```diff
@@ -253,8 +252,8 @@ struct VMCompiler : ExprFunctor<void(const Expr& expr)> {
 
   void VisitExpr_(const MatchNode* match_node) {
     auto match = GetRef<Match>(match_node);
-    LOG(FATAL) << "translation of match nodes to the VM is"
-               << "currently unsupported" << std::endl;
+    LOG(WARNING) << "translation of match nodes to the VM is"
```

Comment:
why this fatal -> warning change? it is indeed unsupported, and by warning doesnt u just make the error more silent?

Reply:
@MarisaKirisame Current vm workflow compiles all functions in prelude before inference. If we fail here we won't be able to run inference until we have pattern match support. I also need this to run unit test. Agree it makes the error less obvious. I can change it to `ERROR`.

Reply:
can you change it to error and disable prelude instead? I think more safety is better.

Reply:
@MarisaKirisame done. thanks